### ISSUE TITLE
[Bug] [SeaTunnel Spark Start V2] deploy-mode need Lowercase letters, But give an Uppercase in --help

### ISSUE THREAD

**Issue by EricJoy2048** [apache/seatunnel#2811]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

params error.

### SeaTunnel Version

dev

### SeaTunnel Config

```conf
*
```

### Running Command

```shell
sh start-seatunnel-spark-connector-v2.sh --config ../config/spark_hive_to_console.conf --deploy-mode CLIENT --master local
```

### Error Exception

```log
deploy-mode: CLIENT is not allowed.
Usage: start-seatunnel-spark-connector-v2.sh [options]
  Options:
    -t, --check       check config (default: false)
  * -c, --config      Config file
  * -e, --deploy-mode Spark deploy mode (values: [CLIENT, CLUSTER])
    -h, --help        Show the usage message
  * -m, --master      Spark master
    -i, --variable    variable substitution, such as -i city=beijing, or -i 
                      date=20190318 (default: [])
```

### Flink or Spark Version

*

### Java or Scala Version

*

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Carl-Zhou-CN** [comment]
I want to give it a try

**Comment by EricJoy2048** [comment]
> I want to give it a try

Ok, Thank you very much.

**Comment by liugddx** [comment]
Assigned to me

**Comment by EricJoy2048** [comment]
> Assigned to me

It's fixed by @Carl-Zhou-CN  already.

Thank you very much.

**Comment by Carl-Zhou-CN** [comment]
closes #2834
